functions/index.js

```diff
@@ -41,3 +43,47 @@ exports.dailycc = functions.https.onRequest(() => {
 		}
 	};
 });
+
+exports.affirmations = functions.https.onRequest((req, res) => {
```

Comment:
Why do you have req and res here?

Reply:
Vibe bruh it's labeled a work in progress 🤣. This is just for postman testing purposes though, will clean up after

Reply:
Hey Darryl, actually keep the req and res. These functions are essentially http requests, so if we don't send a response the function is eventually gonna timeout and show as an error in firebase.